fetch.bs

```diff
@@ -894,13 +894,11 @@ from APIs using <a for=/>fetch</a> that allow control over
 is a <a>byte-case-insensitive</a> match for one of:
 
 <ul class=brief>
- <li>`<code>Content-Disposition</code>`
  <li>`<code>Content-Encoding</code>`
  <li>`<code>Content-Language</code>`
  <li>`<code>Content-Location</code>`
- <li>`<code>Content-Range</code>`
  <li>`<code>Content-Type</code>`
- <li>`<code>ETag</code>`
+ <li>`<code>Content-Length</code>`
```

Comment:
`Content-Length` is a forbidden header name so doesn't need to be listed. It's always computed just before the request, so nulling body does effectively cause it to be omitted.

Reply:
It meets my experiment. Thanks.